collections/requirements.yml

```diff
@@ -0,0 +1,7 @@
+---
+collections:
+  - name: ansible.netcommon
+  - name: ansible.posix
+  - name: community.docker
+  - name: kubernetes.core
+  - name: gluster.gluster
```

Comment:
Not sure about thegluster one as it's in a contrib playbook and not directly kubespray.

Reply:
I  moved gluster.gluster in tests/requirements.yml